identity/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.2.3
```

Comment:
```suggestion
## 0.2.3 - unreleased
```

Reply:
thanks, updated.

Reply:
I will release right away once merged. Thus reverted the change.